_docs/for-developers.md

```diff
@@ -171,3 +171,26 @@ Registering many tools or very few tools?
         - Use the Write API webservice and client.  After some setup time (getting GitHub and Quay.io tokens, setting up service, etc), it allows you to programmatically create GitHub and Quay.io repositories on the fly, then register/publish them on Dockstore.
 
 Generally, Write API webservice and client has the highest setup time compared to the other methods of registering.  But, as you register more tools, the Write API tends to become the better choice (since it performs many intermediary steps for you).
+
+## Converting File-path Based Imports to Public http(s) Based Imports
+
+See https://cromwell.readthedocs.io/en/develop/Imports/ for a general knowledge on imports.  
+
+Imports allow you to reference other files in your workflow.  There are two types of resources that are supported in imports: http(s) and file-path based. Any public http(s) based URL can be used as the resource for an import, such as a website, GitHub, GA4GH compliant TES endpoint, and etc.
+
+There are times when you may want to convert file-path based imports to public http(s) imports.  One such reason is to ensure compatibility with FireCloud since it currently does not support file-path based imports.  There are so many different ways to convert to a public http(s) based import, the following are two examples.
+
+You can host your file on GitHub and import it in the workflow descriptor like Topmed does:
```

Comment:
Maybe `like the Topmed workflow does` or simply `in the workflow descriptor like this:`? Most people will presumably not know what TopMed is.